src/DSharp.Compiler.Tests/DSharp.Compiler.Tests.csproj

```diff
@@ -793,6 +794,9 @@
     <None Include="Source\Expression\Generics3\Code.cs">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Include="Source\Expression\Generics4\Code.cs">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
Any reason this is always and the others PreserveNewest?

Reply:
should be PreserveNewest